src/LaunchDarkly.XamarinSdk/ILdClient.cs

```diff
@@ -240,7 +240,7 @@ public interface ILdClient : IDisposable
         /// As of this version’s release date, the LaunchDarkly service does not support the <c>metricValue</c>
         /// parameter. As a result, calling this overload of <c>Track</c> will not yet produce any different
         /// behavior from calling <see cref="Track(String, LdValue)"/> without a <c>metricValue</c>.
```

Comment:
Whoops - this whole bit (lines 240-242) is obsolete and should be deleted.

Reply:
```suggestion
```